packages/server/lib/environment.js

```diff
@@ -66,6 +66,9 @@ try {
   // https://github.com/cypress-io/cypress/pull/20271
   app.commandLine.appendSwitch('disable-ipc-flooding-protection')
 
+  // ensure we get the most accurate memory usage
+  app.commandLine.appendSwitch('enable-precise-memory-info')
```

Comment:
Should we only enable this launch option when when the experiment is enabled?

Reply:
This was discussed here: https://github.com/cypress-io/cypress/pull/25462#discussion_r1070018967